docs/xks/developer-guide/flux.md

```diff
@@ -0,0 +1,103 @@
+---
+id: flux
+title: Flux
+---
+
+Flux is very tightly coupled with XKF and it's how we deploy all tenant yaml to our clusters.
+The goal of this document is to give you as a developer a quick overview of
+how to get use flux in XKF and perform simple debugging tasks.
+Flux have it's own [official documentation](https://fluxcd.io/) and where you can find much more information.
+
+Before reading any further please read through the [core concepts](https://fluxcd.io/docs/concepts/) of flux.
+
+Our very own Philip Laine [presented](https://www.youtube.com/watch?v=F7B_TBcIyl8) on KubeCon 2021 show a similar workflow that we use.
+
+## CLI
+
+You don't have to use the cli but it can be very helpful especially if want to force a reconcile of a flux resource.
+In some of the commands and debugging we assume that you have the cli installed.
+
+Here you can find more information on how to setup the [flux cli](https://fluxcd.io/docs/installation/).
+
+## XKF and flux
+
+In the XKF framework we talk allot about tenants, from a Kubernetes point of view a tenant is a namespace that have been generated using terraform.
```

Comment:
allot ~ assign
a lot ~ much

Reply:
I think if we mean namespace we should use that word, but I think this text wants to say that a tenant typically is a team and that each tenant is assigned a namespace?